cargo-dist/tests/integration-tests.rs

```diff
@@ -37,16 +37,19 @@ cargo-dist-version = "{dist_version}"
 installers = ["shell", "powershell", "homebrew", "npm", "msi", "pkg"]
 tap = "axodotdev/homebrew-packages"
 publish-jobs = ["homebrew", "npm"]
-targets = ["x86_64-unknown-linux-gnu", "x86_64-apple-darwin", "x86_64-pc-windows-msvc", "aarch64-apple-darwin"]
+targets = ["x86_64-unknown-linux-gnu", "aarch64-unknown-linux-gnu", "x86_64-apple-darwin", "x86_64-pc-windows-msvc", "aarch64-apple-darwin"]
```

Comment:
so one reason we didn't use arm linux in our integration tests is because we need to pay for those runners- is it necessary to use this target to test what you want to test?

Reply:
that can be changed, although it needs to be another linux-gnu target. it looks like there's an i686-unknown-linux-gnu. i can use that instead. :+1: